### ISSUE TITLE
RNN: How to reuse optimal parameters found by gridsearch?

### ISSUE THREAD

**Issue by pabuta88** [unit8co/darts#583]
I use the following command to do gridsearch to find the optimal parameter set for a RNN:

`
best_model = RNNModel.gridsearch(
    series=training_series,
    val_series=validation_series,
    start=0.5,
    parameters=parameters,
    metric=mae,
    reduction=np.mean
)`

From the documentation it's not clear to me how to use the output to actually train the model. Is there a way to easily extract the parameters and feed them into the RNNModel function?

**Comment by dennisbader** [comment]
Hi @pabuta88, 

`gridsearch()` returns a tuple of `(best_model_untrained, best_params)`.

```
best_model_untrained, best_params = RNNModel.gridsearch(series=training_series, val_series=validation_series, start=0.5, parameters=parameters, metric=mae, reduction=np.mean)
```

`best_model_untrained` is basically a new `RNNModel` object that was creating using the `best_params`. This is equivalent to 

```
best_model_untrained = RNNModel(**best_params)
```

You can then train and predict like in the example notebooks with `model.fit(...)` and `model.predict(...)`, see [here](https://unit8co.github.io/darts/examples/05-RNN-examples.html).
